boards/arm/serpente/board_init.c

```diff
@@ -0,0 +1,37 @@
+/*
```

Comment:
I see that board_init.c was commented out of the build file, but is included here?

Reply:
I will drop this file